vision_tests/runVisionTests.py

```diff
@@ -0,0 +1,278 @@
+    ('xor-1080p-u8', 'org.khronos.openvx.xor image:1920,1080,U008 image:1920,1080,U008 image:1920,1080,U008')
+]
+
+#  Popular Video Sizes
+# { 2160p: 3840x2160, 1440p: 2560x1440, 1080p: 1920x1080, 720p: 1280x720, 480p: 854x480, 360p: 640x360, 240p: 426x240 }
+openvxNodeTestConfig = [
+    # absdiff U8 - U8
+    ('absdiff-240p-u8', 'org.khronos.openvx.absdiff image:426,240,U008 image:426,240,U008 image:426,240,U008'),
+    ('absdiff-360p-u8', 'org.khronos.openvx.absdiff image:640,360,U008 image:640,360,U008 image:640,360,U008'),
+    ('absdiff-480p-u8', 'org.khronos.openvx.absdiff image:854,480,U008 image:854,480,U008 image:854,480,U008'),
+    ('absdiff-720p-u8', 'org.khronos.openvx.absdiff image:1280,720,U008 image:1280,720,U008 image:1280,720,U008'),
+    ('absdiff-1080p-u8', 'org.khronos.openvx.absdiff image:1920,1080,U008 image:1920,1080,U008 image:1920,1080,U008'),
+    ('absdiff-1440p-u8', 'org.khronos.openvx.absdiff image:2560,1440,U008 image:2560,1440,U008 image:2560,1440,U008'),
+    ('absdiff-2160p-u8', 'org.khronos.openvx.absdiff image:3840,2160,U008 image:3840,2160,U008 image:3840,2160,U008'),
+    # absdiff S16 - S16
+    ('absdiff-240p-s16', 'org.khronos.openvx.absdiff image:426,240,S016 image:426,240,S016 image:426,240,S016'),
+    ('absdiff-360p-s16', 'org.khronos.openvx.absdiff image:640,360,S016 image:640,360,S016 image:640,360,S016'),
+    ('absdiff-480p-s16', 'org.khronos.openvx.absdiff image:854,480,S016 image:854,480,S016 image:854,480,S016'),
+    ('absdiff-720p-s16', 'org.khronos.openvx.absdiff image:1280,720,S016 image:1280,720,S016 image:1280,720,S016'),
+    ('absdiff-1080p-s16', 'org.khronos.openvx.absdiff image:1920,1080,S016 image:1920,1080,S016 image:1920,1080,S016'),
+    ('absdiff-1440p-s16', 'org.khronos.openvx.absdiff image:2560,1440,S016 image:2560,1440,S016 image:2560,1440,S016'),
+    ('absdiff-2160p-s16', 'org.khronos.openvx.absdiff image:3840,2160,S016 image:3840,2160,S016 image:3840,2160,S016')
+]
+
+# Import arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('--runvx_directory',    type=str, default='',
+                    help='RunVX Executable directory - required')
+parser.add_argument('--hardware_mode',      type=str, default='CPU',
+                    help='OpenVX Vision Function target - optional (default:CPU [options:CPU/GPU])')
+args = parser.parse_args()
+
+runvxDir = args.runvx_directory
+hardwareMode = args.hardware_mode
+
+# check arguments
+if runvxDir == '':
+    print(
+        "ERROR: RunVX Executable Directory Required")
+    exit()
+if hardwareMode not in ('CPU', 'GPU'):
+    print("ERROR: OpenVX Hardware supported - CPU or GPU]")
+    exit()
+
+# RunVX Application
+runVX_exe = runvxDir+'/runvx'
+runvx_exe_dir = os.path.expanduser(runVX_exe)
+
+print("\nrunVisionTests - OpenVX Vision Tests V-"+__version__+"\n")
+os.system('(cd gdfs; mkdir openvx_test_results)')
+for i in range(len(visionTestConfig)):
```

Comment:
Codacy found an issue: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6404702)